tests/testdata/tmp_directory

```diff
@@ -1 +1 @@
-[{'dn': 'cn=alice,ou=example,o=test', 'attributes': {'oid': '2', 'cn': 'alice', 'accountExpires': 131024988000000000, 'homeDirectory': '/home/alice', 'mobile': ['1234', '45678'], 'userPassword': 'alicepw', 'givenName': 'Alice', 'objectGUID': '\xef6\x9b\x03\xc0\xe7\xf3B\x9b\xf9\xcajl\rM1', 'email': 'alice@test.com', 'sn': 'Cooper'}}, {'dn': 'cn=bob,ou=example,o=test', 'attributes': {'oid': '3', 'cn': 'bob', 'accountExpires': 9223372036854775807, 'homeDirectory': '/home/bob', 'mobile': '123456', 'userPassword': 'bobpw\xc3\xa9\xc3\xa9\xc3\xa9', 'givenName': 'Robert', 'objectGUID': '\xef6\x9b\x03\xc0\xe7\xf3B\x9b\xf9\xcajl\rMw', 'email': 'bob@example.com', 'sn': 'Marley'}}, {'dn': 'cn=manager,ou=example,o=test', 'attributes': {'oid': '1', 'cn': 'manager', 'accountExpires': 9223372036854775807, 'mobile': '123354', 'userPassword': 'ldaptest', 'givenName': 'Corny', 'objectGUID': '\xef6\x9b\x03\xc0\xe7\xf3B\x9b\xf9\xcajl\rMT', 'email': 'ck@o', 'sn': 'keule'}}, {'dn': 'cn=frank,ou=sales,o=test', 'attributes': {'oid': '5', 'cn': 'frank', 'accountExpires': 9223372036854775807, 'mobile': '123354', 'userPassword': 'ldaptest', 'givenName': 'Frank', 'objectGUID': '\xef7\x9b\x03\xc0\xe7\xf3B\x9b\xf9\xcajl\rMT', 'email': 'fh@o', 'sn': 'Hause'}}]
```

Comment:
I think, this is a remnant of the test process. I'll remove it in my commit.

Reply:
thx,